src/MarDyn.cpp

```diff
@@ -58,42 +59,42 @@ void initOptions(optparse::OptionParser *op) {
 /**
  * @brief Helper function outputting program build information to given logger
  */
-void program_build_info(Log::Logger *log) {
-	log->info() << "Compilation info:" << std::endl;
+void program_build_info() {
```

Comment:
```suggestion
void log_program_build_info() {
```
The name should reflect more clearly what happens.